samples/edge-inferencing-and-mlops/docs/README.md

```diff
@@ -0,0 +1,18 @@
+# MLOps <!-- omit in toc -->
+
+The goal of this document is to capture the requirements and designs for the MLOps sections of the project.
+
+## Documents  <!-- omit in toc -->
+
+- [AML Data Drift](ml-data-drift.md)
+- [Machine Learning Workload Architecture](ml-workload-architecture.md)
+- [ML Model Orchestration](ml-model-orchestration.md)
```

Comment:
Really comprehensive docs. Do we have any recommendation on how to scale ML model training and management for multiple manufacturing lines, each could have different data and therefore different model (although model code is still same)?

Reply:
We do, we scaled it down for the sharing, but we had data filtering which took into account the lines. I also believe that the models were tagged based on the lines to  differentiate which model to pull for different lines

Reply:
You don't have to add it to this PR, but it will be really nice to add a doc later for this scenario. We struggle every time how to scale ML model for multiple lines.